compiler/test/fail_compilation/must_use_comma.d

```diff
@@ -1,7 +1,9 @@
 /+
 TEST_OUTPUT:
 ---
-fail_compilation/must_use_comma.d(16): Error: ignored value of `@mustuse` type `must_use_comma.S`; prepend a `cast(void)` if intentional
+fail_compilation/must_use_comma.d(18): Error: template `tuple` is not callable using argument types `!()(S, void)`
```

Comment:
Ideally we would issue an error before the typecons error is triggered.

Reply:
Seems one way to fix this test might be to remove the parentheses? I don't think in general we can issue a `mustuse` error if the result of a function call is not used whose argument is `mustuse`.